BlazarUI/app/scripts/components/dashboard/StarredModulesTableRow.jsx

```diff
@@ -29,41 +39,43 @@ class StarredModulesTableRow extends Component {
       );
     }
 
-    const latestBuild = item.builds[0].build;
-    const latestBuildGitInfo = item.builds[0].gitInfo;
-
+    const commitInfo = latestBuild.get('commitInfo');
     let commitMessage;
     let sha;
 
-    if (has(latestBuild, 'commitInfo')){
+    if (commitInfo !== undefined) {
       commitMessage = (
-        <CommitMessage message={latestBuild.commitInfo.current.message} />
+        <CommitMessage message={commitInfo.get('current').get('message')} />
       );
     }
     
-    if (latestBuild.sha !== undefined) {
+    if (latestBuild.get('sha')) {
       const gitInfo = {
-        host: latestBuildGitInfo.host,
-        organization: latestBuildGitInfo.organization,
-        repository: latestBuildGitInfo.repository
+        host: latestBuildGitInfo.get('host'),
+        organization: latestBuildGitInfo.get('organization'),
+        repository: repository
       }
 
+      latestBuild.sha = latestBuild.get('sha');
```

Comment:
yea, I introduced immutable after starting the project, so there's a lot of stuff that consumes regular javascript objects.  if you want a plain object you can just use `latestBuildGitInfo.toJS()`. Or you can take the time to covert over things like the `Sha` component to consume immutable objects. that would be worth doing over time.